src/chart/line/LineChart.tsx

```diff
@@ -45,7 +45,12 @@ const NeoLineChart = (props: ChartProps) => {
   const marginLeft = settings.marginLeft ? settings.marginLeft : 36;
   const marginTop = settings.marginTop ? settings.marginTop : 24;
   const marginBottom = settings.marginBottom ? settings.marginBottom : 40;
-  const lineWidth = settings.lineWidth ? settings.lineWidth : 2;
+  const lineWidth = (settings): number => {
```

Comment:
Can you change this into an inline value instead of a function? Similar to what other settings are doing. Something like :

`Const lineWidth = settings.type == 'scatter' ? 0 : settings.lineWidth || 2;`